Core/GraphQL/GraphQLFiles/sell_getOrders.graphql

```diff
@@ -0,0 +1,7 @@
+query getOrders($pageSize: Int!, $page: Int = 1){
+  me {
+    orders(pageSize:$pageSize page:$page){
```

Comment:
Add a spacing after the closing bracket please? And also add a spacing after `:`